boto3/dynamodb/types.py

```diff
@@ -62,6 +62,9 @@ def __repr__(self):
         return 'Binary(%r)' % self.value
 
     def __str__(self):
+        return self.value if six.PY2 else self.value.decode('utf-8')
```

Comment:
We generally don't like to have `six.PY2` or `six.PY3` hanging around outside of `compat.py`. How about instead we say something like `if isinstance(six.string_types)`?